packages/p2p-v2/src/components/StarRating/__tests__/StarRating.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import StarRating from '../StarRating';
+import { render, screen } from '@testing-library/react';
+
+describe('StarRating', () => {
+    it('should render the correct stars', () => {
+        render(<StarRating initialValue={3} ratingValue={3.3} />);
+        expect(screen.queryAllByTestId('dt_p2p_v2_star_rating_empty_star')).toHaveLength(5);
+        expect(screen.queryAllByTestId('dt_p2p_v2_star_rating_full_star')).toHaveLength(5);
+    });
```

Comment:
@adrienne-deriv what does it means to have length value `5`  for both empty and full star case ? since you passed `initialValue={3}` and `ratingValue={3.3}`, this should affect the value of the stars as well right ?

Reply:
also the test description says `should render the correct stars` , but looking at the assertions in the test case, it's not clear how it fulfils that description.

Reply:
I might need to re-word the test description, it should render our star icons instead

Reply:
`react-simple-star-ratings` renders all of the star icons regardless of the `ratingValue`, it uses SVG clippings to display the ratingValue by covering the empty star icons with the filled star icons

Reply:
it's fine, in this case I think these assertions are not serving any purpose here since it cannot tell if the component is working properly or not, is there any way we can get the state of the rating value ? 

If not, then I think we should add an inline snapshot for this component

Reply:
IMO better we test this during integration testing, there we will see it properly tested, it is hard to grab the values of the Stars since it doesn't really expose it, and you have to count it through the number of stars shown, which is even more work. And since its already at 100% coverage, no need to go the distance